### DISCUSSION TITLE
Update Transmission on Zyxel

### DISCUSSION THREAD

**Discussion by Petrprogs** [transmission/transmission#3523] [General]
Is there a way to update the transmission on Zyxel Keenetic Omni? I'm not waiting for a solution, just maybe someone has come across it.

**Comment by nevack** [comment]
Maybe by using some custom firmware like OpenWRT?
Or maybe there's OPKG/Entware support like on Keenetics?

**Reply by Petrprogs** [reply]
This is my only router, it's a little scary to flash it with custom firmware. Unfortunately, it is impossible to access it via ssh, only web or telnet.
